src/magic_folder/test/cli/common.py

```diff
@@ -38,10 +101,11 @@ def succeeded(self):
         return self.code == 0
 
 @inline_callbacks
-def cli(argv, global_config=None, http_client=None):
+def _run_cli(command, argv, global_config=None, http_client=None):
     """
-    Perform an in-process equivalent to the given magic-folder command.
+    Perform an in-process equivalent to the given command.
```

Comment:
is it worth saying something about the special `RequestTraversalAgent`-handling behavior you also get from this function?